substrate/frame/nomination-pools/src/weights.rs

```diff
@@ -675,59 +677,40 @@ impl<T: frame_system::Config> WeightInfo for SubstrateWeight<T> {
 	/// Proof: `Balances::Locks` (`max_values`: None, `max_size`: Some(1299), added: 3774, mode: `MaxEncodedLen`)
 	fn adjust_pool_deposit() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `901`
+		//  Measured:  `934`
 		//  Estimated: `4764`
-		// Minimum execution time: 70_284_000 picoseconds.
-		Weight::from_parts(71_375_000, 4764)
+		// Minimum execution time: 78_952_000 picoseconds.
+		Weight::from_parts(80_655_000, 4764)
 			.saturating_add(T::DbWeight::get().reads(4_u64))
 			.saturating_add(T::DbWeight::get().writes(2_u64))
 	}
-	/// Storage: `NominationPools::PoolMembers` (r:1 w:0)
-	/// Proof: `NominationPools::PoolMembers` (`max_values`: None, `max_size`: Some(237), added: 2712, mode: `MaxEncodedLen`)
 	fn apply_slash() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `694`
-		//  Estimated: `3702`
-		// Minimum execution time: 13_403_000 picoseconds.
-		Weight::from_parts(14_064_000, 3702)
-			.saturating_add(T::DbWeight::get().reads(1_u64))
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 2_846_000 picoseconds.
+		Weight::from_parts(3_058_000, 0)
 	}
-	/// Storage: `NominationPools::PoolMembers` (r:1 w:0)
-	/// Proof: `NominationPools::PoolMembers` (`max_values`: None, `max_size`: Some(237), added: 2712, mode: `MaxEncodedLen`)
 	fn apply_slash_fail() -> Weight {
 		// Proof Size summary in bytes:
-		//  Measured:  `732`
-		//  Estimated: `3702`
-		// Minimum execution time: 14_419_000 picoseconds.
-		Weight::from_parts(15_004_000, 3702)
-			.saturating_add(T::DbWeight::get().reads(1_u64))
+		//  Measured:  `0`
+		//  Estimated: `0`
+		// Minimum execution time: 2_858_000 picoseconds.
+		Weight::from_parts(3_010_000, 0)
 	}
 	fn pool_migrate() -> Weight {
 		// Proof Size summary in bytes:
 		//  Measured:  `0`
 		//  Estimated: `0`
-		// Minimum execution time: 759_000 picoseconds.
-		Weight::from_parts(819_000, 0)
+		// Minimum execution time: 2_719_000 picoseconds.
+		Weight::from_parts(2_828_000, 0)
```

Comment:
Also surprising that this operation doesn't read or write anything.
The caller is not related to the operation.